CHANGES

```diff
@@ -2,6 +2,13 @@ Version 8.16 (Unreleased)
 -------------------------
 - Added data migration to create UserEmail objects for users whose primary emails did not have them
 - Time series data (used by graphs and other features) is now updated when groups are merged.
+- Added distributions to the release system to better support mobile apps
+
+Schema Changes
+~~~~~~~~~~~~~~
+
+- Added ``Distribution`` model
+- Added ``Event.dist`` column
```

Comment:
This isn't a column being added. This can be ignored for CHANGES. You've just added a field into the `data` dict.